lib/posix-process/exportsyms.uk

```diff
@@ -101,6 +101,10 @@ uk_syscall_e_rt_sigprocmask
 sigsuspend
 uk_syscall_r_rt_sigsuspend
 uk_syscall_e_rt_sigsuspend
+sigtimedwait
```

Comment:
Nitpick but maybe let's put this below `sigwaitinfo` so that it is next to its `uk_syscall_` variants.